### DISCUSSION TITLE
Why is there no "output" folder?

### DISCUSSION THREAD

**Discussion by hb8t** [LSPosed/MagiskOnWSALocal#399] [Q&A]
Under the "\MagiskOnwSALocal\" folder in my Ubuntu system directories, I can only find the "download" folder but not the "output" folder? What to do?

**Comment by AresWeb21** [comment]
I have the following error:

> Extract done

> Calculate the required space
> done

> Expand images
> Convert vhdx to img and remove read-only flag
> ./build.sh: line 135: qemu-img: command not found
> Build: an error has occurred, exit

> Cleanup Work Directory
> Cleanup Download Directory

**Reply by Howard20181** [reply]
Run `scripts/install_deps.sh`

**Reply by AresWeb21** [reply]
> Run `scripts/install_deps.sh`

Thank you! Now works!

**Reply by myusrn** [reply]
Is one time execution of ./scripts/install_deps.sh, dependencies installer ???, a new recently introduced requirement before ./scripts/run.sh as i wasn't finding it was necessary previously?

**Reply by williamvyong** [reply]
i tried to install deps sh but i doesnt find it

**Reply by Howard20181** [reply]
> Is one time execution of ./scripts/install_deps.sh, dependencies installer ???, a new recently introduced requirement before ./scripts/run.sh as i wasn't finding it was necessary previously?

If you meet "command not found" or the download fails due to certificate problems you can run it again.
